README.md

```diff
@@ -1,16 +1,341 @@
+
+### Easy to Replicate
+
+Every time we initialize a new instance of the `Person` class, we create a
+unique object:
+
+```js
+let sarah = new Person('Sarah', 31);
+let evan = new Person('Evan', 34);
+```
+
+Since each instance of `Person` has unique data, our instance methods will
+behave accordingly and only use _their_ data:
+
+```js
+evan.sayHello();
+// => Hello, my name is Evan.
+sarah.sayHello();
+// => Hello, my name is Sarah.
+```
+
+This turns out to be a fantastic help when dealing with many collections of
+similar data. For example, individual records from a server side database can be
+represented as instances of a `class`.
+
+In an interactive application like an online chatroom where multiple people are
+present, you could design a `User` `class` that keeps track of individual
+chatroom users. We can go further, though, and design `class`es for other parts
+of the application. A `User` instance might have many messages in a chatroom, so
+we could create a `Message` `class`. If the application has multiple chatrooms,
+we could create a `Room` `class` to represent every chatroom and any specific
+data about each.
+
+Once we establish the data and methods for a `class`, we can create as many
+copies as we need.
+
+### Improves Understanding by Adding Meaning
+
+By encapsulating related information and behavior, we have organized our code in
+a more meaningful way. It _makes sense_ to us as humans to group related things
+like this. Object Orientation works well for representing real world systems
+and relationships, which makes it easier to comprehend complex code.
+
+```js
+let restaurant = new Restaurant('La Villa', '261 5th Ave, Brooklyn, NY');
+restaurant.addGuest(evan);
+restaurant.serveGuest(evan, new Drink('Water'));
+restaurant.addGuest(sally);
+restaurant.serveGuest(sally, new Drink('Water'));
+evan.sayHello();
+// => Hello, my name is Evan.
+sarah.sayHello();
+// => Hello, my name is Sarah.
+evan.saySmallTalk();
+// => How about this weather?
+```
+
+All `class`es in are structured in a consistent way, so even though we don't
+know the details of the `Restaurant` `class`, we can still infer how the
+`Restaurant`, `Person` and `Drink` `class`es might interact.
+
+When it comes to building functionality into a website, we do tend to deal with
+the abstract, things that are harder to represent conceptually. Object
+Orientation helps here too, though it takes practice to think in the OO mindset.
+For example, we can organize related actions, such as server requests, into a
+single `class`.
+
+One common practice in OO Javascript is to use `class`es to represent different
+DOM elements. Consider what it we do currently to programmatically add a `div`
+to the page:
+
+```js
+let element = document.createElement('div');
+element.className = 'myClassName';
+element.innerHTML = 'This is the text content of the div';
+document.querySelector('body').appendChild(element);
+```
+
+Rewritten as a class could be something like:
```

Comment:
This phrasing is awkward, a more complete sentence could help

Reply:
took this last chunk out to make the lesson a little simpler